webapp/packages/core-ui/src/Form/FormPart.ts

```diff
@@ -174,7 +175,11 @@ export abstract class FormPart<TPartState, TFormState = any> implements IFormPar
     this.state = state;
   }
 
-  protected format(data: IFormState<TFormState>, contexts: IExecutionContextProvider<IFormState<TFormState>>): void | Promise<void> {}
+  protected format(data: IFormState<TFormState>, contexts: IExecutionContextProvider<IFormState<TFormState>>): void | Promise<void> {
+    runInAction(() => {
+      propagateTrim(this.state);
+    });
```

Comment:
I don't like this idea. We have a schema for the state where we can manually specify which fields should be truncated:
https://zod.dev/api?id=strings

Truncating is not always required and may lead to unwanted behaviour, so it can't be applied to the whole state, especially when we have complex states